landing-page/content/common/release-notes.md

```diff
@@ -68,6 +68,15 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
 </dependencies>
 ```
 
+## 1.0.0 release
+
+This release is based on the latest 0.14.1 release. It includes changes to remove deprecated APIs and the following additional bug fixes:
+* Increase metrics limit to 100 columns ([#5933](https://github.com/apache/iceberg/pull/5933))
+* Bump Spark patch versions for CVE-2022-33891 ([#5292](https://github.com/apache/iceberg/pull/5292))
+* Exclude Scala from Spark runtime Jars ([#5884](https://github.com/apache/iceberg/pull/5884))
+
+
+## Past releases
```

Comment:
This also needs to fix the heading level of the 0.14.1 release section.

Reply:
Fixed 👍